### PR TITLE
fix(New Relic AI): making updates for new features

### PR DESCRIPTION
**Pull request by homelessbirds**
Making updates to the New Relic AI doc.

### CODE REVIEW

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 1]
```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:31]: Not part of this PR, but I suggest using a non-cropped screenshot here. The image is hard to parse and the crop makes it seem really "busy" since the background (the charts) have no visual context.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 2]
```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
...
     />    
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
+
+New Relic AI leverages a variety of skills and tools to complete analysis on your behalf, including:
+Natural language to query (NRQL, NerdGraph) translation
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:37]: Hard to parse. Maybe something like:

Transform natural language into queries (NRQL/Nerdgraph)

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 3]
```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
     />    
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
+
+New Relic AI leverages a variety of skills and tools to complete analysis on your behalf, including:
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:36]: I feel a bit weird calling them `skills and tools`.  These are more capabilities of the AI, not skills or tools.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 4]
```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
...
+Natural language to query (NRQL, NerdGraph) translation
+Anomaly detection
+Error log analysis
+Error stack trace analysis
+Docs synthesis
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:41]: Might want to expand on this - not clear what this means and I don't think it's mentioned elsewhere in the doc

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx (hunk 3 above)
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:36]: I'd reframe this entire list. Instead of things it "provides", talk about what the user can do with it:

.... including:

* Transform natural language into queries (NRQL/Nerdgraph)
* Detect anomalies
* Analyze error logs
* Analyze stack traces
* Summarize New Relic platform concepts from docs
* Summarize and analyze dashboards

Comment (bradleycamacho) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:36]: I'd probably rewrite that list though, that was a quick idea

Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:36]: The list isn't formatted correctly, 
Screenshot 2024-01-04 at 4 18 03 PM

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 5]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
         src={askgrok}
     />
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create  the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to createthe right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:55]:
```suggestion
[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 6]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
 3. Scroll down to the **New Relic AI** section and click **Enable**.
 
-## Sample questions [#sample-questions] 
+### Sample questions [#sample-questions] 
 
 You can chat with our AI to better understand every aspect of your data but if you're struggling with where to begin, here are some prompts. 
 
-### Get started with New Relic 
-Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+<Tabs>
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:67]: Our tab component does a really poor job at seperating the "tab" group from the rest of the content on the docs. 

Notice how in this screenshot you can't tell that `Get help analyzing error logs` isn't part of the tab. I'd move the tab group to the very bottom of the doc.

Screenshot 2024-01-04 at 4 19 31 PM

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 7]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
+    <TabsBar>
+    <TabsBarItem id="1">Get started with New Relic</TabsBarItem>
+    <TabsBarItem id="2">Improve response time</TabsBarItem>
+    <TabsBarItem id="3">Take observability to the next level</TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+
+    <TabsPageItem id="1`">
+    Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:77]:
```suggestion
    Our AI chat experience can help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 8]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
+    <TabsPageItem id="3">
+    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
 
 * How can I improve the performance of my infrastructure?
 * Were unique user sessions up or down last week compared to the week before?
+    </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+### Get help analyzing error logs [#errors]
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:100]: The headers in this doc are a bit weird. Since there is quite a bit of content on the doc, it's a bit jarring to have so many h3s in the doc. I'd toy around with fixes (or don't, this is a small issue) but I think just making most of them h2s is fine.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 9]
```diff
@@ -11,12 +11,13 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
...
-  type="youtube"
-  id="XMSUVmNKVyM"
-/>
+import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-error-pop-up.webp'
+
+import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
+
+New Relic AI (NRAI) is designed to help engineers improve observability. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:22]: I'd rewrite some of the header titles. These are just some ideas, but since the doc is long I think being able to skim the headers is really important.

* `Chat with AI` -> `Ask AI questions`
* `NRQL console` -> `Get help creating NRQL queries`
*

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 10]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
 
 * How can I improve the performance of my infrastructure?
 * Were unique user sessions up or down last week compared to the week before?
+    </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+### Get help analyzing error logs [#errors]
+
+Error log lines can be difficult to parse. New Relic AI helps you learn more about errors, query for related information and ask about next steps.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:102]:
```suggestion
Error log lines can be difficult to parse. New Relic AI helps you learn more about errors, query for related information, and ask about next steps. 
```

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 11]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
+
+<img
+    title="explain this error discussion"
+    alt="A screenshot showing New Relic AI explaining an error"
+    src={aiExplainThisErrorPopUp}
+/>
+
+### Analyze stack traces in errors inbox [#stack-traces]
+
+As with the logs management integration, we are bringing helpful generative AI skills directly into popular New Relic troubleshooting and root cause analysis interfaces like errors inbox.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:130]: I'd rewrite this. Seems very markety.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 12]
```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
...
+    title="explain this error discussion"
+    alt="A screenshot showing New Relic AI explaining an error"
+    src={aiExplainThisErrorPopUp}
+/>
+
+### Analyze stack traces in errors inbox [#stack-traces]
+
+As with the logs management integration, we are bringing helpful generative AI skills directly into popular New Relic troubleshooting and root cause analysis interfaces like errors inbox. 
+
+With a single click, you can leverage New Relic AI to explain an error stack trace and suggest possible next steps.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:132]: Delete or rewrite (serves same role as sentence above)

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:31]: Yes, I agree. I am waiting on better marketing imaging. This is just a placeholder @bradleycamacho .

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 13]
```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
...
 
-* What are some recent incidents in my system?
-* Which infrastructure hosts are showing signs of degradation over the past hour?
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** > **Logs**.
+2. Click **All logs**.
+3. Search for `errors`.
+2. Click on any resulting log lines and look for the **Explain this error** button at the top of the log details panel. Our AI will provide a summary of the error. 
 
-### Take observability to the next level 
-Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:71]: @homelessbirds since "Black Friday" is only in the United States, maybe you should mention that to help set the context for non-US readers.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 14]
```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
...
-/>
+import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-error-pop-up.webp'
+
+import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
+
+New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
 
 We help you get to the heart of your system by offering two quick and easy ways to ask questions: AI chat, and the NRQL console.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:24]: It seems like boiling the options down to two approaches might be too simplistic since you're offering four different experiences:

* Ask AI questions
* Get help creating NRQL queries
* Get help analyzing error logs
* Analyze stack traces in errors inbox

Maybe you could just skip the number of ways and let the headings show users how they might interact with New Relic AI?

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 15]
```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
...
+* Leverage our docs to understand New Relic platform concepts
 
 Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store, and you start receiving alerts that response times for your `Checkout-service` are lagging. If you don't have time to dig into the issue, you can just ask New Relic AI "What's on fire?" or "What should I handle first today?" New Relic AI can tell you what's broken in your system and help you explore data with charts and tables.
 
-### NRQL console [#NRQL-console]
+### Get help creating NRQL queries [#NRQL-console]
+
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 <img
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:43]: I'm not seeing an NRAI button in this screen shot. Also, you'll want to remove "Grok" references in the image properties.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 16]
```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
...
-* How can I improve the performance of my infrastructure?
-* Were unique user sessions up or down last week compared to the week before?
+* How often is this error happening? Meaning, what's the error rate? Is it 1% or 50%?
+* What hosts are throwing these errors? I would like to know is it across all of my hosts or is it just affecting one or two of them?
+
+### Analyze stack traces in errors inbox [#stack-traces]
+
+Using our AI to analyze stack traces will reduce context switching and help speed up triage and analysis. 
+
+<img
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:86]: @homelessbirds I think you should obfuscate the various instances of "Host" and "URL Host" that appear in the doc. Just hide enough to make it more secure.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 17]
```diff
@@ -85,6 +118,41 @@ New Relic AI is becoming more accurate and responsive the more it's used. We wan
 
 Also, New Relic AI can recognize multiple languages and respond in them. If you have use cases for multi-lingual support, please test them with our AI and give us feedback.
 
+New Relic AI responses are AI-generated suggestions. While efforts are made to keep information and responses up to date, suggestions may not always be accurate, complete, or error-free. We would like to encourage you to have a human verify any information provided by our AI using other sources before any action is taken. AI's suggestions are provided without any warranties or guarantees of accuracy.
 
+## Sample questions [#sample-questions]
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:123]: @homelessbirds what would you think of moving this section so it follows the section about where you can interact with NRAI? It seems like readers are going to mostly be focused on the usage and not the security and how AI continues to build accuracy.

**Review by rhetoric101 — COMMENTED** [review]
@homelessbirds this looks good! I inserted a few suggestions for you to consider.

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:43]: It's part of the NRQL console but I can highlight it in green!

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx (hunk 16 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:86]: Done

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:43]: Done

File: src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx [hunk 18]
```diff
@@ -7,71 +7,141 @@ tags:
...
-    />    
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+New Relic AI provides a variety of capabilities to help you:
+* Transform natural language into queries (NRQL/Nerdgraph)
+* Detect anomalies
+* Analyze error logs
+* Understand stack traces
+* Leverage our docs to understand New Relic platform concepts
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx:35]: I like these, very action oriented

**Review by akristen — APPROVED** [review]
